frontend/packages/ceph-storage-plugin/src/components/create-storage-system/payloads.ts

```diff
@@ -1,11 +1,13 @@
+/* eslint-disable no-console */
```

Comment:
Disable line based  where required with a comment on why you're disabling it.

Reply:
I think it was by miss.

Reply:
Please fix it.